src/llm/ollama_client.py

```diff
@@ -3,23 +3,24 @@
 
 from src.logger import Logger
 
+logger = Logger()
```

Comment:
Initiate the logger inside the class `__init__`?

Reply:
list_models is a staticmethod, can't pass self in it

Reply:
Based on `list_models` only usage [here](https://github.com/stitionai/devika/blob/523cbcf709b21a385c91d928a37b20568a8c38f6/src/llm/llm.py#L30), it should also work as a classmethod. In that case, these global variables could be moved into the class (as static members).